src/Mod/Draft/draftmake/make_ellipse.py

```diff
@@ -60,22 +60,28 @@ def make_ellipse(majradius, minradius, placement=None, face=True, support=None):
     support : 
         TODO: Describe.
     """
+
     if not App.ActiveDocument:
         App.Console.PrintError("No active document. Aborting\n")
         return
+
     obj = App.ActiveDocument.addObject("Part::Part2DObjectPython", "Ellipse")
     Ellipse(obj)
+
     if minradius > majradius:
         majradius, minradius = minradius, majradius
     obj.MajorRadius = majradius
     obj.MinorRadius = minradius
     obj.Support = support
+
+    if face != None:
```

Comment:
Probably should be changed to `if face is not None:`  
ref: https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Reply:
Thanks I'll try to remember this next time.